configs/components/fesom/fesom-2.1.yaml

```diff
@@ -201,52 +200,47 @@ restart_out_sources:
         fesom_bin_restart_info: fesom_bin_restart/*.info
         fesom_bin_restart: fesom_bin_restart/np${nproc}/*
         wiso_restart: fesom.${end_date!syear}.wiso.restart.nc
-        icb_restart: iceberg.restart #.${parent_date!syear}
-        icb_restart_ISM: iceberg.restart.ISM
+        icb_restart: iceberg.restart.${end_date!syear} #_${start_date!syear!smonth!sday}-${end_date!syear!smonth!sday} #.${parent_date!syear}
+        icb_restart_ISM: iceberg.restart.ISM.${end_date!syear} #_${start_date!syear!smonth!sday}-${end_date!syear!smonth!sday}
 
 outdata_sources:
         fesom: "*.fesom.*.nc"
 outdata_targets:
-        fesom: "*.fesom.*${start_date!smonth}.${start_date!sday}.nc"
+        #fesom: "*.fesom.*${start_date!smonth}.${start_date!sday}.nc"
+        fesom: "*.fesom.*.nc"
```

Comment:
This is tricky, this will change the behaviour of FESOM output writting for everyone... such a change could be a pain, specially for people running monthly and daily simulations. I understand we are renaming the output from FESOM... not an easy question to answer, given that many many users have a say. I would say, you guys can make a decision about this in Paleo, and then you can put it as part of AWIESM.yaml so that it doesn't conflict with other people using fesom.yaml? or even in the runscript? The other option would be to change it for everyone here, as you are suggesting.